python/ray/tune/logger.py

```diff
@@ -70,6 +72,8 @@ def on_result(self, result):
 
 
 class JsonLogger(Logger):
+    LOG_FILE_PATTERNS = ("*/result.json", )
```

Comment:
Can we use the `result.py` constant strings?

Reply:
I think it might be nice to tie these to the logger classes. This is useful especially if a user implements their custom loggers.